src/components/PageHeader.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './PageHeader.module.css'
+
+/**
+ * Generic page header component used by all pages
+ */
+const PageHeader = ({ children }) => (
+    <h2 className={styles.header} data-test="page-header">
```

Comment:
Yeah, I'm not 100% sure about this approach, but I think that was my thinking. I had a look at some other DHIS2 apps and saw `h2` being used as the main "page title", so figured that was wise. But thinking about it, perhaps having the page title being `h1` makes more sense. I'm open to changing it if you think that would be better?

Reply:
I think that makes sense. I looked at this and tried to read up on what is best practice regards to screen readers. There was a bit of conflicting information (surprise :eyeroll:), but I guess it seems like making the title text in the header bar the `<h1>` would align most with the [mdn guidance](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

> A page should generally have a single `<h1>` element that describes the content of the page (similar to the document's [<title> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title))

In which case, all of these would be `<h2>` like you have them

I guess we should ideally make the change to header bar so that we are not confusing screen readers by skipping over header levels.

Reply:
Ok, thanks for a taking a look! I'll keep them as `h2` here and make a note to investigate updating the header bar (I have an ongoing investigation into screen readers and DHIS2 apps, so this helps).